js/src/components/Sidebar.js

```diff
@@ -45,6 +45,7 @@ export default function Sidebar( { settings, store, theme } ) {
 						<StoreProvider store={ store }>
 							<SeoAnalysis
 								shouldUpsell={ settings.shouldUpsell }
+								shouldUpsellWordFormRecognition={ settings.shouldUpsellWordFormRecognition }
```

Comment:
I'm not seeing the upsell notice in the sidebar any longer. I guess this should be `settings.isWordFormRecognitionActive`.